protocol/0042-LIQF-setting_fees_and_rewarding_lps.md

```diff
@@ -146,6 +146,33 @@ An existing LP has `average entry valuation 1090.9` and `S=110`. Currently the s
 (average entry valuation) = 1090.9
 ```
 
+### Calculating supplied liquidity from probability of trading 
+
+At every vega time change calculate the supplied liquidity provided by each committed LP. 
+This is done by taking into account all the volume they're providing between the tightest price monitoring bound and then 
+use the formula provided by [probability weighted liquidity measure](./0034-PROB-prob_weighted_liquidity_measure.ipynb).
```

Comment:
Just to clarify - the supplied liquidity is calculated from both persistent limit orders AND any volume from the pegged orders created by the liquidity provision?

If so the supplied liquidity can be greater than the committed liquidity? I know this was already the case, but before an LP gained nothing from supplying more liquidity than they committed, now they will (albeit less efficiently).

Reply:
I've added a line in the text.